docs/tutorials/t-08.md

```diff
@@ -0,0 +1,127 @@
+## Objective
+
+This tutorial is to demonstrate [`Algosigner`](https://github.com/PureStake/algosigner) and [`Wallet Connect`](https://walletconnect.com/) in webapp using [`@algo-builder/web`](https://github.com/scale-it/algo-builder/tree/master/packages/web). [`@algo-builder/web`](https://github.com/scale-it/algo-builder/tree/master/packages/web) is designed to be used with web dapps as well as scripts and user programs and also allows you to interact with contracts easily.
+
+## What is `@algo-builder/web`?
+
+This package provides a class `WebMode` which has variety of high level functions like, [`waitForConfirmation`](algobuilder.dev/api/web/classes/web.html#waitForConfirmation), [`executeTx`](<(algobuilder.dev/api/web/classes/web.html#executeTx)>), [`signTransaction`](<(algobuilder.dev/api/web/classes/web.html#signTransaction)>), etc. These functions help sending transactions and building dapps.
+
+## Requirements
+
+- Node 14+
+- Clone [`wallet-connect-template`]().[TODO: Add template URL when template PR is merged.]. This is a Vue3 app.
+- `pipenv` with our `Pipfile`. Read more about [`pipenv`](#Using-Pipenv).
+- Yarn `v3.1+` or NPM `v8.0+` or PNPM `v6.21+`.
+- Setup the accounts on [`Algosigner`](https://github.com/PureStake/algosigner) and [`Wallet Connect`](https://walletconnect.com/) wallet. It could be MainNet, TestNet or private net.
+
+## Steps
+
+1. Setup the [`wallet-connect-template`](). [TODO: Add template URL when template PR is merged.]
+2. The above template should be accessbile on [`localhost`](http://localhost:8080/) and you shoulb be able to see below screen with dropdown.
```

Comment:
```suggestion
2. The above template should be accessbile on your localhost with port 8080 and you should be able to see below screen with drop down.
```

Reply:
Or you can do with better way as you refer ❤️

Reply:
I would prefer to link the localhost URL so that the user can directly access it without having to enter into the browser. I have committed the other suggestion apart from this.